### PR TITLE
Push notifications example

### PR DESCRIPTION
**Pull request by cmjarreau**
Starter push notification example

Screen Shot 2023-05-10 at 11 21 19 AM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,19 @@ and `@lifeomic/react-native-sdk` (and its peer dependencies), you can have an
...
+
+Add Firebase to your android project:
+https://firebase.google.com/docs/android/setup
+
+#### iOS
+
+Allow push notification entitlement/capabilities through XCode:
+https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns
+
+After generating your certificates for either platform, you can collaborate with
```
Comment (cmjarreau) [inline README.md:35]: not exactly sure how the end user will reach out to us and if any information on that point should be included here

File: package.json [hunk 2]
```diff
@@ -43,7 +43,8 @@
     "react-native-toast-message": "^2.1.6",
     "react-native-vector-icons": "^9.2.0",
     "react-native-webview": "^11.26.1",
-    "react-query": "^3.39.3"
+    "react-query": "^3.39.3",
+    "rn-emoji-keyboard": "^1.2.2"
```
Comment (evanheisler) [inline package.json:47]: This package doesn't appear to be used at all.

File: src/App.tsx [hunk 3]
```diff
@@ -28,6 +30,16 @@ function App() {
     <DeveloperConfigProvider
       developerConfig={{
         simpleTheme,
+        additionalNavigationTabs: [
+          {
+            name: 'NotificationsExample',
+            component: NotificationsExampleScreen,
+            options: {
+              tabBarLabel: t('tabs-notificatiosn', 'Notifications Playground'),
```
Comment (evanheisler) [inline src/App.tsx:38]: nit: typo in the translation key

File: src/utils/constants.ts
Comment (evanheisler) [inline src/utils/constants.ts:1]: This seems unnecessary. These strings aren't translated and are for demo purposes anyway. In your example component you have a bunch of static text so may as well keep it consistent.

File: package.json (hunk 2 above)
Comment (cmjarreau) [reply package.json:49]: @namick @evanheisler whats the best way / practice that we have to handle upstream/downstream dependencies?

Comment (evanheisler) [reply package.json]: Right now even if the SDK version doesn't change, a fresh `yarn install` is going to grab the latest minor/patch and want to install peer dependencies. It makes it a bit confusing in CR but not a big deal for now — a commit comment may help pointing to the SDK change. We could consider pinning the SDK package version. Then peer dependencies would be seen alongside the SDK version bump.

**Review by evanheisler — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (markdlv) [reply README.md:35]: This is perfect!

**Review by markdlv — APPROVED** [review]

**Pull request merged**